### ISSUE TITLE
Window size of second Coteditor window

### ISSUE THREAD

**Issue by scoskey** [coteditor/CotEditor#1350]
### Description

In #1025 people discuss that the default window size is the same as the last opened window. However currently (at least for me) there seems to be a bug with the window size of the second-opened Coteditor window not being the same size as the first. More generally, opening several files, closing, re-opening, seems to yield at least two distinct sizes.

### To Reproduce

1. Double click a file to open it in Coteditor. The window size is as expected.
2. Command+O and select a second file and open it. The window size is different than expected.

### Expected behavior

If the default window width and height are left blank, and the user does not resize any windows, all windows should open to the same size (same as last opened window).

### CotEditor version

4.3.3

### macOS version

12.5.1

### Additional context

I have carefully positioned my window size and font size to be good for my own editing.

**Comment by 1024jp** [comment]
Thank you for the feedback. This is definitely a regression in CotEditor 4.3.3.

Everything stems from a bug in macOS that autosaving window size & position is not restored property (This is a long-standing issue...). I added a workaround in CotEditor 4.3.3 but it seems not working in some cases.

So I've just revised my code again and I think now it works fine.
I'll release CotEditor 4.3.4 that fixes this issue pretty soon.
